test/applyRules.test.js

```diff
@@ -1,3 +1,6 @@
+/**
+ * @jest-environment jsdom
+ */
```

Comment:
Should this be removed now? It seems like the tests are still passing so I'm guessing we can keep it?